src/libs/kata-types/src/annotations/mod.rs

```diff
@@ -950,6 +954,16 @@ impl Annotation {
                     KATA_ANNO_CFG_VFIO_MODE => {
                         config.runtime.vfio_mode = value.to_string();
                     }
+                    KATA_ANNO_CFG_SANDBOX_BIND_MOUNTS => {
+                        let args: Vec<String> = value
+                            .to_string()
+                            .split_ascii_whitespace()
```

Comment:
Maybe `split_whitespace()` is better. And maybe collect Vec<&str> is also ok, so we will not use `.map` here